### DISCUSSION TITLE
[help] I have opened port but still can't add peers/can't synchronize

### DISCUSSION THREAD

**Discussion by rangerov** [Chia-Network/chia-blockchain#3968] [Support]
Anyone having same issue? 

Chia Nodes | PORT | BLOCK
5.189.201.93 | 8444 | [LOCAL*]
188.233.133.89 | 8444 | [240524]
84.106.156.127 | 8444 | [240523]
81.217.130.233 | 8444 | [240524]

introducer-apne.chia.net:8444
introducer-apse.chia.net:8444

introducer-or.chia.net:8444
introducer-va.chia.net:8444
introducer-eu.chia.net:8444

Tried all of these, nothing can be added to other peers. 

Success: I can see your service on **.***.**.** on port (8444)
Your ISP is not blocking port 8444

**Comment by Leozolt** [comment]
at what point does tracert (windows) / traceroute (linux) fail probing for port 8444

example: traceroute node-eu.chia.net -p 8444

P.S. dont post your public IP here.. If the packet fails on the way you have some info to ask your ISP for assistance. If the packet fails before reaching your public IP gateway there is something wrong with your local network.

**Comment by rangerov** [comment]
i didnt post my ip, i censored myself. this is tracert result
15     *        *        *     Request timed out.
16     *        *        *     Request timed out.
17     *        *        *     Request timed out.
18     *        *        *     Request timed out.
19     *        *        *     Request timed out. 
20     *        *        *     Request timed out.
21   158 ms   144 ms   152 ms  chia.keva.app [5.189.201.93]

there are some failing like this and before that everything is normal

**Comment by chiabug** [comment]
Those timeouts are likely just normal for tracert given that it did get to the destination.

**Comment by Leozolt** [comment]
So can you confirm that you cannot telnet to chia.keva.app (or other nodes) on port 8444 but the traceroute reaches chia.keva.app? (hop 21, the request timing out are just the respective hubs/switches/routers not answering to ICMP request.)

What are the results for traceroute "node hostname or IP" -T -p 8444 ?

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
